environment.yml

```diff
@@ -0,0 +1,17 @@
+name: occ
+channels:
+  - conda-forge # pythonocc-core
+  - defaults
+dependencies:
+  - python=3.8
+  - anaconda
+  - pip
+  - COMPAS
+#  - compas_view2 # could not find a channel that hosts this
+  - pythonocc-core
+  - cython # compas_view2, really
+  - python.app # compas_view2, really
```

Comment:
this will not work and is not needed on windows

Reply:
ah shoot, indeed, so not sure whether the [win] / [not win] selectors work in the `environment.yml` -- but this is a stopgap for `compas_view2` anyways... just a stopgap for now...